frontend/package.json

```
@@ -13,7 +13,8 @@
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
     "format": "prettier --write .",
-    "format:check": "prettier --check ."
+    "format:check": "prettier --check .",
+    "json": "ajv -s pbtar_schema.json -d src/data/scenarios_metadata.json"
```

```suggestion
    "json:check": "ajv -s pbtar_schema.json -d src/data/scenarios_metadata.json"
```

NIT: The command that will be run would be `npm run json:check` which seems more self-evident to what it is doing than `npm run json`

---

done in 18b22b615748602712dfa5dc32d881c58f937cf7